src/Tests/dotnet-workload-install.Tests/GivenDotnetWorkloadInstall.cs

```diff
@@ -420,6 +420,20 @@ public void GivenWorkloadInstallItWarnsWhenTheWorkloadIsAlreadyInstalled()
             Directory.GetFiles(installRecordPath).Count().Should().Be(2);
         }
 
+        [Theory]
+        [InlineData("--verbosity:minimal")]
+        [InlineData("--verbosity:normal")]
+        public void HideManifestUpdatesWhenVerbosityIsMinimalOrNormal(string verbosityFlag)
```

Comment:
Ideally we should also have a test that checks that the AdManifestUpdated message is displayed when the verbosity is detailed or diagnostic.  It may be hard to mock things up so that there are updates available though.